docs/conf.py

```diff
@@ -120,15 +133,15 @@
     # The paper size ('letterpaper' or 'a4paper').
     #
     # 'papersize': 'letterpaper',
-
+    #
     # The font size ('10pt', '11pt' or '12pt').
     #
     # 'pointsize': '10pt',
-
+    #
     # Additional stuff for the LaTeX preamble.
     #
     # 'preamble': '',
-
+    #
```

Comment:
"black" tried to remove empty rows between commented code - so I commented these empty rows to leave file well-formatted.